pyogrio/tests/conftest.py

```diff
@@ -8,7 +8,7 @@
 
 
 _data_dir = Path(__file__).parent.resolve() / "fixtures"
-ALL_EXTS = [".shp", ".gpkg", ".json"]
+ALL_EXTS = [".shp", ".gpkg", ".geojson", ".geojsonl", ".geojsons"]
```

Comment:
Is it needed to have both extensions in here? Because that means we duplicate tests for GeoJSONSeq for all tests that are parametrized.  
Just for testing that the extension is auto-recognized, I think we could also have one targeted test, and here only have one extension for this?

Reply:
True. I removed the ".json" ext in ``ALL_EXTS`` for the same reason, but apparently wasn't quite awake when I added both GeoJsonSeq exts...